src/provider/provider_level_zero.c

```diff
@@ -307,6 +327,18 @@ umf_result_t umfLevelZeroMemoryProviderParamsSetMemoryType(
     return UMF_RESULT_SUCCESS;
 }
 
+umf_result_t umfLevelZeroMemoryProviderParamsSetDeviceOrdinal(
+    umf_level_zero_memory_provider_params_handle_t hParams,
+    uint32_t deviceOrdinal) {
+    if (!hParams) {
+        LOG_ERR("Level zero memory provider params handle is NULL");
```

Comment:
and a minor thing - pls make `Zero` uppercase here

Reply:
Done

Reply:
thx!